packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -10,6 +10,7 @@
  * governing permissions and limitations under the License.
  */
 
+import type {DirectorySymbolType} from '@react-aria/dnd';
```

Comment:
urgh, hopefully this is ok to have in the types package, otherwise I'm a bit at a loss at how to define the `directorySymbol` in such a way that the `@react-types/shared` package doesn't depend on the `@react-aria/dnd` package. 

Alternatively I could make `acceptedDragTypes` accept generic `symbol` instead or move `DroppableCollectionProps` definition out of types and into the `react-aria/dnd` package

Reply:
Annnd circular dependency